### DISCUSSION TITLE
Voice INPUT and NMEA2000

### DISCUSSION THREAD

**Discussion by zuchi238** [canboat/canboat#421] [Ideas]
"am I on course" , "what is my speed"

.. These are the things I want to ask my boat.

So Im trying (with the help of my best friend ChatGPT) to write a small python script to make this happen.

Who is interested?

Bart

**Comment by tkurki** [comment]
If I were you I would use https://github.com/SignalK/signalk-server instead of "raw" canboat. You gain 
- compatibility with both NMEA 0183 and NMEA 2000 input data
- if you package your thing as a plugin a distribution and update mechanism (plugins can be python)
- a wider community for your thing from the get go

(I am heavily involved with Signal K, that just happens to use canboat)

**Comment by zuchi238** [comment]
Hi there, thanks for your input. Will give it a try.
(Would be cool to talk to your boat ... ;)

Bart
